src/eduid_idp/tests/test_SSO.py

```diff
@@ -70,7 +70,7 @@ def make_SAML_request(class_ref):
 
 def _transport_encode(data):
     # encode('base64') only works for POST bindings, redirect uses zlib compression too.
-    return ''.join(data.split('\n')).encode('base64')
+    return b64encode(''.join(data.split('\n')))
```

Comment:
This will fail in PY3, independently of whether data is a str or a bytes object:
```>>> b64encode(''.join(data.split('\n')))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: a bytes-like object is required, not 'str'`